### PR TITLE
LCD-11263 added hosting-with-reserved-env-var

### PR DESCRIPTION
**Pull request by yijutuan**

### CODE REVIEW

**Comment by yijutuan** [comment]
will be using this in a functional test for https://issues.liferay.com/browse/LCD-11263

File: hosting-with-reserved-env-var/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Deployment Activity Tests
```
Comment (christieyoo) [inline hosting-with-reserved-env-var/README.md:1]: Can you change this to `# hosting-with-reserved-env-var`? Other than that, everything else LGTM.

**Review by christieyoo — CHANGES_REQUESTED** [review]

**Review by christieyoo — APPROVED** [review]

**Comment by yijutuan** [comment]
@christieyoo oops, i clicked rereview because I didn't see that it updated with your approval already. You're so fast haha

**Comment by yijutuan** [comment]
@christieyoo Is there a certain number of approvals required? Or should I just wait for someone on the QA team to help me merge this in?

**Review by tainguyen100 — APPROVED** [review]

**Comment by christieyoo** [comment]
I can merge it now!

**Pull request merged**